examples/example_export.py

```diff
@@ -0,0 +1,364 @@
+# -*- coding: utf-8 -*-
+"""
+Example script showing how to use the LaceworkClient class.
+"""
+
+import csv
+import logging
+from datetime import datetime, timedelta
+
+from dotenv import load_dotenv
+from laceworksdk import LaceworkClient
+
+logging.basicConfig(level=logging.INFO)
+
+load_dotenv()
+
+ISO_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+
+class ReferenceLookup:
+    def __init__(self, key, field, dict, multivalue=False):
+        self.key = key
+        self.field = field
+        self.dict = dict
+        self.multivalue = multivalue
+
+    def lookup(self, value, default=None):
+        # only return the first matching result or default value
+        dict = list(filter(lambda x: x[self.key] == value, self.dict))
+
+        rows = []
+        for row in dict:
+            # return the entire row
+            if self.field is None:
+                rows.append(row)
+            else:
+                for i in self.field.split("."):
+                    if i in row:
+                        row = row[i]
+                    else:
+                        row = default
+                    rows.append(row)
+
+        # return all multiple values
+        if self.multivalue:
+            return rows
+
+        # return first value only
+        else:
+            return rows.pop() if len(rows) > 0 else default
+
+
+class DataHandler:
+    def __init__(self, format, file_path="export.csv"):
+        if format not in ["csv", "dict"]:
+            raise Exception(
+                f"Unsupported export format, exepcted csv or dict found: {format}"
+            )
+
+        self.format = format
+        self.file_path = file_path
+
+    def __open(self):
+        if self.format == "csv":
+            self.header = False
+            self.fp = open(self.file_path, "w")
+            self.writer = csv.writer(self.fp, quoting=csv.QUOTE_ALL)
+            self.dataset = csv.reader(self.fp)
+        else:
+            self.dataset = []
+
+    def __close(self):
+        if self.format == "csv":
+            self.fp.close()
+
+    def insert(self, row):
+        if self.format == "csv":
+            if not self.header:
+                self.writer.writerow(row.keys())
+                self.header = True
+
+            self.writer.writerow(row.values())
+        elif self.format == "dict":
+            self.dataset.append(row)
+
+    def get(self):
+        return self.dataset
+
+    def __enter__(self):
+        self.__open()
+
+        return self
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        self.__close()
+
+
+def query(
+    client,
+    type,
+    object,
+    start_time=datetime.now() + timedelta(days=-1),
+    end_time=datetime.now(),
+    filters=[],
+    returns=None,
+):
+
```

Comment:
It probably doesn't matter for this example, but having a list/dict as a default value is an anti-pattern (https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html)

I would probably do the same for the start_time and end_time too, and given that we always expect UTC, you'll want to explicitly set that.  Add `timezone` to your `datetime` imports and then do something like this:
```suggestion
    start_time=None,
    end_time=None,
    filters=None,
    returns=None,
):

    if start_time is None:
        start_time = datetime.now(timezone.utc) + timedelta(days=-1)
    if end_time is None:
        end_time = datetime.now(timezone.utc)
    if filters is None:
        filters = []

```